src/Modules/Shared/Microsoft.PowerShell.Host/Microsoft.PowerShell.Host.psd1

```diff
@@ -10,5 +10,5 @@ FunctionsToExport = @()
 CmdletsToExport="Start-Transcript", "Stop-Transcript"
```

Comment:
@adityapatwardhan, your last commit had 50 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Validate Update-Help from the Web for one PowerShell module..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for one PowerShell module for user scope..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate 'Save-Help -DestinationPath for one PowerShell modules..Validate Save-Help for the 'Microsoft.PowerShell.Core' module
```powershell
Expected 'Microsoft.PowerShell.Core_00000000-0000-0000-0000-000000000000_en-US_HelpContent.cab', because Save-Help for  should download 'Microsoft.PowerShell.Core_00000000-0000-0000-0000-000000000000_en-US_HelpContent.cab', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 333
333:     $compressedFile | Should -Be $expectedCompressedFile -Because "Save-Help for $module should download '$expectedCompressedFile'"
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'CimCmdlets' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```